scripts/freshvenvs.py

```diff
@@ -124,6 +155,28 @@ def _versions_fully_cover_bounds(bounds: typing.Tuple[str, str], versions: typin
     return versions[0] >= Version(upper_bound)
 
 
+def _iterate_venvs(venv: riotfile.Venv, package: str) -> bool:
```

Comment:
Given the use of `package` on line 165, it seems to me that the second parameter to this function is treated as a "suite name" rather than a "package name". Maybe `suite_name` would be a more descriptive name?